app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,76 @@
+class RecipesController < ApplicationController
+  before_action :set_recipe, only: %i[show edit update destroy]
+
+  # GET /recipes or /recipes.json
+  def index
+    @user = current_user
+    @recipes = @user.recipes.includes(:user)
```

Comment:
- Just like for foods, a logged in user should be able to see all recipes irrespective of the owner of the recipe, please fix this 🙏. Also remember that only the owner of a recipe can delete the recipe.